content/_index.md

```diff
@@ -26,7 +26,7 @@ OSM runs an Envoy based control plane on Kubernetes, can be configured with SMI
 ## Install
 
 ### Prerequisites
-- Kubernetes cluster running Kubernetes v1.18.0 or greater
+- Kubernetes cluster running Kubernetes v1.19.0 or greater
```

Comment:
Are you basing this on what's supported in terms of k8s versioning? Should OSM have a version skew policy published somewhere that we then link here? https://github.com/openservicemesh/osm#prerequisites still says 1.18, so we shouldn't change it without dev team buy-in.

Reply:
moved to #111